### PR TITLE
Admin message settings backend added

### PR DESCRIPTION
**Pull request by SaptakS**
#1665 

Things done:-

* [x] If both checkboxes in a row are activated for a message, the message should show up in both ways and the user gets an email. 
* [x] Please implement that message either show up or not (depending on the admin settings) in the mail settings of the user

@mariobehling @niranjan94 review

### CODE REVIEW

File: app/helpers/data_getter.py [hunk 1]
```diff
@@ -600,12 +601,26 @@ def get_page_by_id(page_id):
...
+        results = Page.query.filter_by(url=url)
         if results:
             return results.one()
         return results
 
     @staticmethod
+    def get_all_message_setting():
+        settings_list = MessageSettings.query.all()
+        all_settings = {}
+        for index, d in enumerate(settings_list):
```
Comment (SaptakS) [inline app/helpers/data_getter.py]: unavoidable

File: app/helpers/data_getter.py [hunk 2]
```diff
@@ -600,12 +601,26 @@ def get_page_by_id(page_id):
...
+            all_settings[settings_list[index].action] = {'mail_status': settings_list[index].mail_status,
+                                                         'notif_status': settings_list[index].notif_status,
+                                                         'user_control_status': settings_list[index].user_control_status}
```
Comment (SaptakS) [inline app/helpers/data_getter.py]: unavoidable

File: app/views/admin/super_admin/messages.py [hunk 3]
```diff
@@ -1,15 +1,27 @@
...
         return self.render(
             '/gentelella/admin/super_admin/messages/messages.html',
             mails=MAILS,
-            notifications=NOTIFS
+            notifications=NOTIFS,
+            message_settings=message_settings
         )
+
+    @expose('/update', methods=['POST'])
+    def update_view(self):
```
Comment (SaptakS) [inline app/views/admin/super_admin/messages.py]: unavoidable.

File: app/models/message_settings.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+EVENT_ROLE = 'Event Role Invitation'
+SESSION_ACCEPT_REJECT = 'Session Accept or Reject'
+SESSION_SCHEDULE = 'Session Schedule Change'
+EVENT_PUBLISH = 'Event Published'
+AFTER_EVENT = 'After Event'
+
+
+class MessageSettings(db.Model):
+    __tablename__ = 'message_settings'
+    id = db.Column(db.Integer, primary_key=True)
```
Comment (SaptakS) [inline app/models/message_settings.py]: This is the convention we are using in every table, changing will create a lot of issues right now.

**Comment by SaptakS** [comment]
In the naming conventions, we are following `snake_case` everywhere right now instead of `camelCase`. So that is something we need to decide on and refactor on the entire code.

**Comment by mariobehling** [comment]
Let's follow up about this. Maybe Codacy could be configured to use other conventions. Nevertheless, the build is failing currently.

**Comment by SaptakS** [comment]
@mariobehling this PR is ready to be merged. I have already written the tests for the modules necessary.

**Comment by rafalkowalski** [comment]
@SaptakS maybe you can change d variable to __. take a look on https://www.codacy.com/app/fossasia/open-event-orga-server/pullRequest?prid=289039

**Comment by SaptakS** [comment]
@rafalkowalski Not sure.. I think it will still treat it as a variable.. But I will surely check that out..

**Pull request merged**